collector/compile-benchmarks/type-system-chess/perf-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "category": "secondary",
+    "artifact": "library"
+}
```

Comment:
Since this is not interesting for codegen, we could run it only for `check` to save time, using
```json
"excluded_profiles": ["Debug", "Opt", "Doc"]
```

Reply:
Makes sense to me. I can confirm that `check` and `build` take the same amount of time.